app/models/katello/content_view_filter.rb

```diff
@@ -157,6 +157,10 @@ def validate_content_view
       if self.content_view.composite?
         errors.add(:base, _("cannot contain filters if composite view"))
       end
+
+      if self.content_view.import_only?
+        errors.add(:base, _("cannot add filter to import-only view"))
```

Comment:
Can you add test for this

Reply:
See changes in test/models/content_view_filter_test.rb